source/tutorial/create-system-dsn.txt

```diff
@@ -12,41 +12,161 @@ Create a System DSN
    :depth: 1
    :class: singlecol
 
-The following steps describe how to create a system Data Source
-Name (DSN) for the BI Connector's :binary:`~bin.mongosqld`
-process. A DSN is a saved configuration which
-describes a database connection to be
-used by an :abbr:`ODBC (Open Database Connectivity)` driver.
-Once the DSN is created for the |bi-short|, you can configure a wide
-range of SQL clients and BI tools to use the DSN and import
+.. _Create a keytab file: https://web.mit.edu/kerberos/krb5-latest/doc/admin/admin_commands/kadmin_local.html#ktadd
+.. _Kerberos Principals: https://web.mit.edu/kerberos/krb5-1.5/krb5-1.5.4/doc/krb5-user/What-is-a-Kerberos-Principal_003f.html
+.. _Heimdal: https://www.h5l.org
+.. _Heimdal configuration file: https://www.h5l.org/manual/HEAD/info/heimdal/Configuration-file.html#Configuration-file
+
+The following steps describe how to create a system |dsn| for the |bi-short|\'s :binary:`~bin.mongosqld`
+process. A |dsn| is a saved configuration which
+describes a database connection for an |odbc| driver to use.
+Once the |dsn| is created for the |bi-short|, you can configure a wide
+range of SQL clients and BI tools to use the |dsn| and import
 data from MongoDB.
 
-.. COMMENT This directive is just to render a tab strip for the prereqs
-.. tabs-platforms::
+Prerequisites
+-------------
+
+Before creating a |dsn|, you should:
+
+.. tabs::
```

Comment:
For a real life discussion:

I don't think this content belongs on this page because it has nothing to do with the client side of things.  This page is all about configuring the client.

Reply:
Understood, but all of these prerequisites need to be configured or the client cannot connect to anything.